druntime/src/core/stdc/config.d

```diff
@@ -260,10 +260,176 @@ else
 }
 
 // ABI layout of native complex types.
-private struct _Complex(T)
+struct _Complex(T)
```

Comment:
Why is this not `private` anymore? A `public` symbol should have a DDoc comment.

Reply:
Eventually I'd like to see std.complex derive from this rather than duplicating it.

Reply:
Added a Ddoc comment.